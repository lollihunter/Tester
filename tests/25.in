20 3 20
1 1
10 2
11 3
2 3
3 1
17 3
2 2
8 1
15 1
6 2
19 3
7 3
19 2
11 2
7 1
19 1
12 3
9 2
17 2
20 3
