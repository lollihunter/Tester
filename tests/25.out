12
1 5 10 12 18 2 14 17 9 19 13 20 
